### PR TITLE
drtprod: add datadog-agent setup for workload-chaos

### PR DESCRIPTION
**Pull request by itsbilal**
This change updates the drtprod script to add the workload-choas datadog-agent setup commands and config files.

Epic: none

Release note: None

### CODE REVIEW

File: scripts/drtprod
Comment (sudomateo) [inline scripts/drtprod:118]: Are there many metrics exported from these endpoints? I ask because these metrics will be billed as custom metrics [per the documentation](https://docs.datadoghq.com/integrations/openmetrics/) and a high volume of custom metrics can have unintended cost implications. If there's a reference that will list the metrics exported from these endpoints we could link it here as a comment.

**Review by sudomateo — CHANGES_REQUESTED** [review]
Changes look good just blocking merge until we discuss the custom metrics.

File: scripts/drtprod
Comment (itsbilal) [reply scripts/drtprod:118]: We're talking about on the order of magnitude, ~50 metrics per endpoint. That seems like it's above our default allocation, so i'll bring this down a little by eliminating the go_* ones that we don't care about. Thanks for pointing that out!

**Review by sudomateo — APPROVED** [review]

**Comment by itsbilal** [comment]
TFTR!

Had to replace the regex with `tpcc_*` and `kv_*` because the workload prefix was getting trimmed by the raw_metric_prefix attribute. But now it works. Will merge after green CI - thanks!

**Comment by itsbilal** [comment]
bors r=sudomateo

**Pull request merged**
